source/partials/drupal-9/troubleshooting.md

```diff
@@ -0,0 +1,57 @@
+## Pantheon Launch Check Status Error: services.yml does not exist
+
+After you set up Drupal 9, the following error might be displayed in the **Best practices** section of the Pantheon Launch Check:
+
+> <span  style="color:red">x <strong>sites/default/services.yml:</strong></span> services.yml does not exist! Copy the default.service.yml to services.yml and see https://www.drupal.org/documentation/install/settings-file for details.
+><br />
+><br />
+>
+> *Create services.yml file inside sites/default directory by copying default/services.yml file. Refer to https://www.drupal.org/documentation/install/settings-file for details.*
+
+Ensure your site's [Development Mode](/guides/quickstart/connection-modes/) is set to **Git**, use the terminal on the local machine where you cloned the site, and from the project's root directory:
+
+1. Copy `default.services.yml` to `services.yml`:
+
+ ```bash{promptUser: user}
+ cp web/sites/default/default.services.yml web/sites/default/services.yml
+ ```
+
+1. Commit and push:
+<!-- need to provide example w/out web? -->
+ ```bash{promptUser: user}
+ git add web/sites/default/services.yml && git commit -m "init services.yml"
+ git push origin master
+  ```
+
+Learn more about the [service configuration](/services-yml#create-and-modify-servicesyml) file.
+
+## Error: Class ViewPageController Does Not Exist
+
+As reported in [Drupal Issue 3161309](https://www.drupal.org/project/drupal/issues/3161309), some fresh installations may encounter the error:
```

Comment:
If we're migrating/upgrading Drupal, this bug should not apply. Also, suggest to reinstall Drupal is definitely not an option here. I'd consider removing this item

Reply:
removed